featuretools/selection/selection.py

```diff
@@ -127,6 +132,19 @@ def remove_highly_correlated_features(feature_matrix, features=None, pct_corr_th
     fm_to_check = (feature_matrix[features_to_check]).select_dtypes(
         include=numeric_and_boolean_dtypes)
 
+    if include_categorical:
```

Comment:
I would refactor this out, and explain what it does in one or two sentences at the docstring.

Reply:
You mean drop the flag completely and always check correlation of categorical features? It's ok if we never plan to merge back into public featuretools repo, otherwise it'll change the behavior of dependent users

Reply:
No, I meant:
```if include_categorical:
        _add_label_encoded_categorical_features_to_check(fm_to_check, ....)

Reply:
Done